scripts/project-setup-scripts/terra-billing-project-owner.yaml

```diff
@@ -0,0 +1,15 @@
+title: "terra_billing_project_owner"
+description: "permissions for billing project owners"
+stage: "GA"
+includedPermissions:
```

Comment:
I've pruned this list in several passes, getting progressively more aggressive each pass. See commit history for details

Reply:
I am inclined to start with the yolo level and start with some communication that we are tightening security and users can contact us if we are too aggressive... adding permissions back in is easy